docs/guides/managed-backup/dedicated-backup-namespace-auto-backup/index.md

```diff
@@ -0,0 +1,407 @@
+
+## Before You Begin
+
+- At first, you need to have a Kubernetes cluster, and the `kubectl` command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+- Install `Stash` in your cluster following the steps [here](/docs/setup/README.md).
+
+- You should be familiar with the following `Stash` concepts:
+  - [BackupConfiguration](/docs/concepts/crds/backupconfiguration.md)
+  - [BackupSession](/docs/concepts/crds/backupsession.md)
+  - [Repository](/docs/concepts/crds/repository.md)
+
+Here, we are going to take backup from the `prod-1`, `prod-2` and `prod-3` namespaces and manage the backup from `backup` namespace using Auto Backup.
+
+Let's create the above-mentioned namespaces,
+
+```bash
+$ kubectl create ns prod-1
+namespace/prod-1 created
+
+$ kubectl create ns prod-2
+namespace/prod-2 created
+
+$ kubectl create ns prod-3
+namespace/prod-3 created
+
+$ kubectl create ns backup
+namespace/backup created
+
+```
+
+>**Note:** YAML files used in this tutorial can be found [here](https://github.com/stashed/docs/guides/managed-backup/dedicated-backup-namespace-auto-backup/examples).
+
+## Prepare Backup Blueprint
+
+In this section, we are going to prepare our Backup Blueprint object.
+
+### Prepare Backend
+
+We are going to use [GCS Backend](/docs/guides/backends/gcs.md) to store the backed up data. You can use any supported backend you prefer. You just have to configure Storage Secret and `spec.backend` section of `BackupBlueprint` to match your backend. To learn which backends are supported by Stash and how to configure them, please visit [here](/docs/guides/backends/overview.md).
+
+> For GCS backend, if the bucket does not exist, Stash needs `Storage Object Admin` role permissions to create the bucket. For more details, please check the following [guide](/docs/guides/backends/gcs.md).
+
+**Create Storage Secret:**
+
+At first, let's create a Storage Secret for the GCS backend in the `backup` namespace,
+
+```bash
+$ echo -n 'changeit' > RESTIC_PASSWORD
+$ echo -n '<your-project-id>' > GOOGLE_PROJECT_ID
+$ mv downloaded-sa-json.key GOOGLE_SERVICE_ACCOUNT_JSON_KEY
+$ kubectl create secret generic -n backup gcs-secret \
+    --from-file=./RESTIC_PASSWORD \
+    --from-file=./GOOGLE_PROJECT_ID \
+    --from-file=./GOOGLE_SERVICE_ACCOUNT_JSON_KEY
+secret/gcs-secret created
+```
+
+### Prepare RBAC
+
+Stash does not grant necessary RBAC permissions to the backup job for taking backup from a different namespace. In this case, we have to provide the RBAC permissions manually. This helps to prevent unauthorized namespaces from getting access to a database via Stash.
+
+**Create ServiceAccount:**
+
+At first, we are going to create a ServiceAccount in the `backup` namespace. This ServiceAccount will refer to the granted permissions for taking backup. Here is the YAML of the ServiceAccount,
```

Comment:
We will grant necessary RBAC permissions to this `ServiceAccount` and use it in the backup job.